### ISSUE TITLE
Figcaption color does not work with section backgrounds

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#4455]
# Problem
Figcaption has a set color that doesn't adjust to section backgrounds in layout builder. 

# Steps to reproduce

- https://sandbox.prod.drupal.uiowa.edu/image-captions

# Proposed solution

- Find usages of `figcaption` in uids_base and remove color property and adjust opacity to .7.
